### DISCUSSION TITLE
Chatify Messaging Customization

### DISCUSSION THREAD

**Discussion by enanwangai** [munafio/chatify#210] [Q&A] [answered]
Hi, thank you for this wonderful package that is simple and straightforward. 
I would need your help given that I am just familiarizing myself with Laravel. 
1. I wanted to know how to automatically show the contact list on loading of the chatify page with their last correspondence kind of like what we see in other chat platforms. Mine unfortunately does not show that and I haven't seen documentation for the same. 
2. How to turn the Chatify messages into events to be displayed as notifications via Pusher on the home page. 

Kindly assist me.
Regards.

**Comment by munafio** [comment, marked as answer]
Hi @enanwangai ,
to answer your questions:
1. Contacts in Chatify will show you the users you've contacted with (messaging each other), if it is not as you want then simply you can change this because Chatify gives you flexibility and everything can be customized as your needs.
2. Chatify messages are events sent from the backend to the frontend via Pusher SDK, you can listen on the event required and use it as a notifications for the home page.

**Reply by enanwangai** [reply]
Thank you for that response. 
Do you have any documentation for the first question? i.e. Contacts in Chatify. Because I can't see the threads of those previously chatted with. especially once there is a refresh. 
For the second one I am sorted, thanks.

**Reply by enanwangai** [reply]
![image](https://user-images.githubusercontent.com/112321535/187187125-c02622d0-81bf-4a80-9c16-89f3840c955d.png)
![image](https://user-images.githubusercontent.com/112321535/187187197-8500706c-adea-4950-a6b8-000020fa4025.png)
You see, the reason I ask is that I have a couple of users this logged-in user has been conversing with but for some reason, the system isn't displaying their chat once the page is refreshed. 

I am sorry, to disturb you. I humbly ask for your assistance. 
Thank you very much!!

**Reply by munafio** [reply]
The user did not talk with you, then it won't appear in a contact list, only will added if you both have contacted each other. As I told you before, you can change this behavior to yours since Chatify is fully customizable.

**Reply by munafio** [reply]
If you already contacted each other and still not displayed in the contact list, then you have to check if any errors occurred there .. check the console and network tab..etc
